diff --git a/triage_issue_27901/src/Controller/Issue27901Controller.php b/triage_issue_27901/src/Controller/Issue27901Controller.php
index a0c2813..2d103dd 100644
--- a/triage_issue_27901/src/Controller/Issue27901Controller.php
+++ b/triage_issue_27901/src/Controller/Issue27901Controller.php
@@ -2,10 +2,13 @@
 
 namespace App\Controller;
 
+use Doctrine\DBAL\Connection;
+
 class Issue27901Controller
 {
-    public function test()
+    public function test(Connection $connection)
     {
-
+        $qb = $connection->createQueryBuilder();
+        $connection->executeUpdate($qb);
     }
 }
\ No newline at end of file
