diff --git a/symfony/src/Symfony/Bundle/SecurityBundle/Security/TargetPathHelper.php b/symfony/src/Symfony/Bundle/SecurityBundle/Security/TargetPathHelper.php
new file mode 100644
index 0000000..6c6a949
--- /dev/null
+++ b/symfony/src/Symfony/Bundle/SecurityBundle/Security/TargetPathHelper.php
@@ -0,0 +1,11 @@
+<?php
+
+namespace Symfony\Bundle\SecurityBundle\Security;
+
+class TargetPathHelper
+{
+    public function savePath(string $uri)
+    {
+        
+    }
+}
