diff --git a/src/Symfony/Bundle/SecurityBundle/Tests/Security/TargetPathHelperTest.php b/src/Symfony/Bundle/SecurityBundle/Tests/Security/TargetPathHelperTest.php
index 2271302..d54f610 100644
--- a/src/Symfony/Bundle/SecurityBundle/Tests/Security/TargetPathHelperTest.php
+++ b/src/Symfony/Bundle/SecurityBundle/Tests/Security/TargetPathHelperTest.php
@@ -36,4 +36,4 @@ class TargetPathHelperTest extends TestCase
         $targetPathHelper = new TargetPathHelper($session, $firewallMap, $requestStack);
         $targetPathHelper->savePath('/foo');
     }
-}
\ No newline at end of file
+}
